main.go

```diff
@@ -2,13 +2,15 @@ package main
 
 import (
 	"context"
+	goerrors "errors"
 	"os"
 	"os/signal"
 	"runtime/debug"
 	"syscall"
 
 	"github.com/pkg/errors"
 
+	dpotelgo "github.com/ONSdigital/dp-otel-go"
```

Comment:
it strikes me that the dpotelgo here is probably unnecessary - I think that's what it's called anyway, so omitting it should work

Reply:
Removed alias